packages/create-next-app/helpers/get-pkg-manager.ts

```diff
@@ -1,19 +1,10 @@
-export type PackageManager = 'npm' | 'pnpm' | 'yarn' | 'bun'
+import type { PackageManager } from '../types'
 
 export function getPkgManager(): PackageManager {
-  const userAgent = process.env.npm_config_user_agent || ''
-
-  if (userAgent.startsWith('yarn')) {
-    return 'yarn'
-  }
-
-  if (userAgent.startsWith('pnpm')) {
-    return 'pnpm'
-  }
-
-  if (userAgent.startsWith('bun')) {
-    return 'bun'
-  }
-
+  const userAgent = process.env.npm_config_user_agent
+  if (!userAgent) return 'npm'
```

Comment:
Lets keep the `process.env.npm_config_user_agent || ''` so we dont need two different `return npm`.